CONTRIBUTING.md

```diff
@@ -33,15 +33,15 @@ Tecnócratas Digitales.
 
 * Elección de artículo.
     * Comenta de qué va a tratar.
-    * Asegurate que el artículo aporta valor.
+    * Asegúrate que el artículo aporta valor.
 * Redacción del artículo.
-    * Utiliza material propio.
-    * Si te basas en datos y artículos, añádelos en la sección de Bibliografía después del artículo.
+    * Utiliza material propio en la medida de lo posible.

```

Comment:
Por favor, revisad esta puntualización y marcad con un 👍 si estáis de acuerdo o con un 👎 si no lo estáis explicando las razones y/o proponiendo una sugerencia en una repuesta.

Reply:
Yo obligaría a poner una sección al final con fuentes oficiales, ¿Por qué lo has retirado?

Reply:
No está retirado... Sólo te he puesto el foco en la línea 38. Lo que pasa es que como corregí conflictos puede que los diffs no se vean completos... Si abres el fichero verás que sí que está y no está quitado. 😉

Reply:
Vale, parecia que se quitaba y no se ponia por el snippet.